client/my-sites/github-deployments/deployments/styles.scss

```diff
@@ -106,6 +106,7 @@
 
 	&__commit-details {
 		display: flex;
+		white-space: normal !important;
```

Comment:
Weird, why do we need the important here?

Reply:
The td text wrap is overriding it. but we don't want to make the change to all td else the date would wrap also.

Reply:
Oh I see. Then we could've extended https://github.com/Automattic/wp-calypso/pull/88053/files#diff-951eea2cf6eaa251e4a2ee09fd52e9af0f437678fbc8d4dc7d4498bf017f3662R52. But I guess it's no biggie.